src/core/jit/snapshots/tailcall__core__jit__builder__tests__resolving_operation-2.snap

```diff
@@ -44,27 +45,31 @@ expression: plan.into_nested()
                         id: 1,
                         name: "id",
                         type_of: ID!,
+                        parent_type: "Post",
```

Comment:
We are storing the parent type for performance reasons? Can't we perform a simple lookup and find the parent information in O(1) complexity?

Reply:
not exactly. The type of the parent could be different to this property because it could contain interface or union variant whilst type of the parent is some concrete type/union. 

perhaps, different naming could help? like `on_type`?

Reply:
Understood. Lets rename it to `type_condition` add a bit of documentation. 

More: https://spec.graphql.org/October2021/#sec-Type-Conditions

Reply:
done